src/foraging_gui/GenerateMetadata.py

```diff
@@ -152,17 +152,19 @@ def _mapper(self):
                 'NINDS': 'NINDS',
                 'Simons Foundation': 'SIMONS',
             },
-            'ephys_rig_behavior_daq_names':[
+            'rig_daq_names_janelia_detector':[
                 "Harp Behavior",
                 "Harp Sound",
                 "Harp clock synchronization board",
                 "Harp sound amplifier",
                 ],
-            'behavior_rig_behavior_daq_names':[
+            'rig_daq_names_new_detector':[
```

Comment:
Can we call it "aind_detector" and not "new" since that will get confusing as time goes one

Reply:
updated!